LunchTime/Views/Home/Brno.cshtml

```diff
@@ -0,0 +1,43 @@
+﻿@{
```

Comment:
This view is totally same as Olomouc (except title). I would recommend to you create one view or partial view.

Reply:
Ok why not, I can merge this pages together and create some more generic part. also the HomeController has lot of same code ;-). this can be also improved